src/components/deposit/GetAddress.js

```diff
@@ -33,27 +27,22 @@ const GetAddress = ({ status, getBitcoinAddress }) => {
         <div className="description">
           {statusText}
         </div>
+        <div className="error">
+          { btcAddressError }
+        </div>
       </div>
     </div >
   )
 }
 
 const mapStateToProps = (state, ownProps) => {
   return {
-    status: state.deposit.invoiceStatus
+    status: state.deposit.invoiceStatus,
+    btcAddressError: state.deposit.btcAddressError,
```

Comment:
Above and below we bind to a simple `error` variable, but here we name it completely. We should do one or the other consistently, IMO.